hal/lib/athena/SPI.cpp

```diff
@@ -16,8 +16,8 @@
 #include "HAL/cpp/priority_mutex.h"
 #include "spilib/spi-lib.h"
 
-static_assert(sizeof(uint32_t) <= sizeof(void*),
-              "This file shoves uint32_ts into pointers.");
+static_assert(sizeof(int32_t) <= sizeof(void*),
+              "This file shoves int32_ts into pointers.");
```

Comment:
I think these asserts can actually be removed, as we don't cast ints into pointers anymore. I'll do that in a separate PR